csharp/test/Microsoft.ML.OnnxRuntime.EndToEndTests.Capi/C_Api_Sample.cpp

```diff
@@ -57,13 +57,13 @@ int main(int argc, char* argv[]) {
   // print number of model input nodes
   status = OrtSessionGetInputCount(session, &num_input_nodes);
   std::vector<const char*> input_node_names(num_input_nodes);
-  std::vector<size_t> input_node_dims;  // simplify... this model has only 1 input node {1, 3, 224, 224}.
+  std::vector<int64_t> input_node_dims;  // simplify... this model has only 1 input node {1, 3, 224, 224}.
                                         // Otherwise need vector<vector<>>
 
   printf("Number of inputs = %zu\n", num_input_nodes);
 
   // iterate over all input nodes
-  for (int i = 0; i < num_input_nodes; i++) {
+  for (unsigned int i = 0; i < num_input_nodes; i++) {
```

Comment:
Why it's unsigned int?

Reply:
Switched back to int. Was initially getting warnings about signed/unsigned comparison -- but can't reproduce them.

Reply:
Why don't use size_t?

Reply:
that works too. updated.